LitvischenkoYevhen/HW5/HW5_task_3.py

```diff
@@ -0,0 +1,6 @@
+# calculate factorial
+num = int(input('Enter the number: '))
+sum = 1
+for i in range(1,num+1):
```

Comment:
good job
but look at this version of code
input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)